eng/pipelines/build.yml

```
@@ -66,8 +66,8 @@ stages:
               DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
               DOTNET_CLI_TELEMETRY_OPTOUT: 1
               DOTNET_MULTILEVEL_LOOKUP: 0
-          - pwsh: ./eng/CodeChecks.ps1
-            displayName: "Check if code is up-to-date"
+          - pwsh: ./eng/SharedCodeCheck.ps1
+            displayName: "Check if shared source codes are up-to-date"
```

```suggestion
            displayName: "Check if shared source code is up-to-date"
```
I think code is uncountable when it refers to source code.